practice.rb

```diff
@@ -0,0 +1,66 @@
+def print_array(a)
+  print "["
+  l = a.length
+  a.each do |item|
+    print "#{item}"
+    if item != a[l-1]
+      print ", "
+    end
+  end
+  print "]"
+end
+
+print_array([4,5,6,4,2,123,42])
+
+def stretch(list)
+  print "["
+  l = (list.length) * 2
+  index = 0
+  list.each do |item|
+    if item % 2 == 0
+      print "#{item/2}, #{item/2}"
+    end
+    if item % 2 == 1
+      print "#{item/2 + 1}, #{item/2}"
+    end
+    index += 1
+    if index < l
+      print ", "
+    end
+  end
+  print "]"
+end
```

Comment:
Your logic is all solid here, but I didn't want anything printed. I wanted a new array returned. Make sure to follow specifications closely.